packages/host/app/components/operator-mode/code-submode/spec-preview.gts

```diff
@@ -183,11 +177,6 @@ class SpecPreviewContent extends GlimmerComponent<ContentSignature> {
   @service private declare realm: RealmService;
   @service private declare operatorModeStateService: OperatorModeStateService;
 
-  constructor(owner: Owner, args: ContentSignature['Args']) {
-    super(owner, args);
-    this.initializeCardSelection();
-  }
```

Comment:
Here is the one of the issues. Is that the selection was made at constructor (by id). And the selection never re-rendered after that unless you interacted with the dropdown. 

This cause things like the LHS panel selection to never work